VisualC/glfont/glfont.vcxproj

```diff
@@ -113,6 +113,7 @@
       <PreprocessorDefinitions>WIN32;NDEBUG;_WINDOWS;HAVE_OPENGL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <RuntimeLibrary>MultiThreadedDLL</RuntimeLibrary>
       <WarningLevel>Level3</WarningLevel>
+      <AdditionalIncludeDirectories>K:\ProjectsMine\SDL_ttf\external\SDL\include</AdditionalIncludeDirectories>

```

Comment:
Please remove these system-specific lines.
These visual studio projects assume you have the SDL and SDL_ttf repositories checked out next to each other (and built SDL before).
See lines 99-100 (which probably should use `ProjectDir` instead as we do in [SDL3_net](https://github.com/libsdl-org/SDL_net/blob/b2ca02cfd9e0fe5c05bd6b0523c779b35e76772c/VisualC/SDL_net.vcxproj#L86-L101)